tests/tpch/generate-data.py

```diff
@@ -1,25 +1,80 @@
+import functools
 import pathlib
 import tempfile
+import warnings
 
 import botocore.session
 import click
+import coiled
 import dask
 import duckdb
 import psutil
+import pyarrow.compute as pc
+from dask.distributed import wait
 
 
 def generate(
-    scale: int = 10, partition_size: str = "50 MiB", path: str = "./tpch-data"
+    scale: int = 10,
+    partition_size: str = "50 MiB",
```

Comment:
Maybe we should change this default up a bit.  "128 MIB" or "256 MiB" maybe?  Is this size on disk or size in RAM?

Or is there some specific rationale for the current value?

Reply:
Unrelated to the current PR though.  Happy to leave for future work.

Reply:
128MB sounds like a good default generally speaking.

Reply:
Super, set to 128, and regenerating the scale-N-strict datasets now. Assuming that's good, I'll regenerate the others as well. (the ones inside 'tpc-h' directory)

Reply:
Nvm, I take it back about generating the other datasets. @fjetter said there will likely be some testing on partition sizing. So as it stands right now, the `scale-N-strict` datasets in `tpc-h` are using 128MiB and the non strict has been left alone (I think 50MiB target). Until we converge on a proper sizing.

Reply:
I've replaced scale-100 already.  I'm ok not doing a ton of testing/tuning on this.